ExomeDepth/make_BEDdetail.py

```diff
@@ -18,9 +18,16 @@ def make_merge_dic(merge_samples):
 
     return merge_dic
 
+def is_child(sampleid):
+    if any([child_tag in sampleid for child_tag in ['CM', 'CF', 'CO']]):
+        return True
+
+def is_parent(sampleid):
+    if any([parent_tag in sampleid for parent_tag in ['PM', 'PF', 'PO']]):
+        return True
```

Comment:
Do you need both? If a sample is not a child -> it is a parent?

Reply:
Could be an option, but them the otherway around. If not parent than child: Parent need to be as 'clean' as possible as these are 'unaffected'. 
But to be sure, I think we should keep both for now, and exclude samples that have unknown status (i.e. typos / research/ validation samples).
We could change this if we can query status from a db.